py/desispec/zcatalog.py

```diff
@@ -1,15 +1,429 @@
-## This script is based on this discussion - https://github.com/desihub/desispec/issues/1355
-## Version: January 21, 2022
+"""
```

Comment:
I haven't seen the test results yet, but the format of the docstrings in this module don't appear to be particularly standard.  Be prepared for test failures due to unrecognized docstring style.

Reply:
@weaverba137, can you please specifically point what the issues are with the docstring? Or you can send me some link with standard docstring example?